### DISCUSSION TITLE
process 'git' exited with code 128

### DISCUSSION THREAD

**Discussion by AlbertJ-314** [lean-dojo/LeanCopilot#102] [Q&A] [answered]
**I encountered problem when running "lake exe LeanCopilot/download", it says "process 'git' exited with code 128".**
**I'm using Mac 15 Air, M3 chip, and my lean-toolchain version is v4.10.0-rc2 so I set the LeanCopilot version to be v1.4.1.**

**Pictures of my terminal.**
<img width="1388" alt="1" src="https://github.com/user-attachments/assets/84715e23-069b-4c42-a13e-d40d747489c9">
<img width="1014" alt="2" src="https://github.com/user-attachments/assets/7ac74549-ed93-4996-958e-cc269bd7ed25">
<img width="1603" alt="3" src="https://github.com/user-attachments/assets/1c77e82e-bb51-4451-b9ba-695130d598c6">

**And I ran lake build despite the "process 'git' exited with code 128", here follows the picture of my VScode after that.**
<img width="1663" alt="4" src="https://github.com/user-attachments/assets/6640cc83-018e-4c86-b227-9957aea229ff">

**Please help!!!**

**Comment by Peiyang-Song** [comment, marked as answer]
It seems that this error suggests that one may not have access to the Internet and the git process. Is this the case for you?

**Comment by Peiyang-Song** [comment]
Combining this discussion with Discussion #101 , which discusses the need for Internet access when running Lean Copilot.
